src/reanimated2/screenTransition/swipeSimulator.ts

```diff
@@ -0,0 +1,228 @@
+'use strict';
+import type {
+  PanGestureHandlerEventPayload,
+  ScreenTransitionConfig,
+  RNScreensTurboModuleType,
+} from './commonTypes';
+import { applyStyle } from './styleUpdater';
+
+const RNScreensTurboModule: RNScreensTurboModuleType =
+  global.RNScreensTurboModule || {
+    startTransition: (_stackTag) => {
+      'worklet';
+      console.warn('[Reanimated] RNScreensTurboModule has not been found.');
```

Comment:
1. Let's come up with a more actionable warning message here with proper troubleshooting.
2. Do we need to have the same warning in all three methods instead of just the first one that gets called?

Reply:
1. 🫡
2. I included a factory that generates a `noop` function. Although the warning solely in the `startTransition` might be sufficient, having this factory already in place could add an extra layer of safety. It's a bit of an overkill, but I believe it's harmless 😅. If you disagree, I can certainly make the change.